### ISSUE TITLE
Question about the parameter gamma ($r$)

### ISSUE THREAD

**Issue by sanyouwu** [qinenergy/adanet#5]
Hi, thanks for your contribution.
I have a question about the parameter gamma in your paper (5), how to set the value of gamma.
In your code, I think it corresponding to : loss = nll_loss + additional_loss.
Which means gamma equals to  1 ? @qinenergy

**Comment by qinenergy** [comment]
Hi! Sorry for the late reply.
Thank you for your interest in our work. 
Yes, the weight for adversarial loss is 1 in all experiments. We updated some relevant details in the new arXiv version for clarification.
